CMakeLists.txt

```diff
@@ -202,7 +202,11 @@ option(ADD_GDB_INDEX_FOR_GOLD "Add .gdb-index to resulting binaries for gold lin
 
 if (NOT CMAKE_BUILD_TYPE_UC STREQUAL "RELEASE")
     # Can be lld or ld-lld or lld-13 or /path/to/lld.
-    if (LINKER_NAME MATCHES "lld")
+    if (LINKER_NAME MATCHES "ld64.lld")
```

Comment:
Could we first list the more common `lld`, followed by the exotic `ld64`?

EDIT: `-Wl` is a no-op ... you might as well remove l. 206 and 207 completely.

Reply:
I tried like this before. `ld64.lld` MATCHES `lld`, so I still had to add `NOT LINKER_NAME MATCHES "ld64"`